dashboard/config/routes.rb

```diff
@@ -94,6 +89,13 @@
       end
     end
 
+    # Data docs are off of curriculum builder as of fall 2022.
+    get 'docs/concepts/data-library', to: 'data_docs#index'
+    get 'docs/concepts/data-library/:key', param: :key, constraints: {data_doc_key: /#{CurriculumHelper::KEY_CHAR_RE}+/o}, to: 'data_docs#show'
+
+    # There are still old curricula that live on curriculum.code.org.
+    # There are also old levels that point to docs on curriculum.code.org.
```

Comment:
nice updates to comments! 👍 

interestingly, it looks like 90+% of the remaining references to CB docs are pointing to specific IDEs, which are now served by rails via the above routes, and there are just a few pointing to `/docs/csd/...` which look like reference guides that we have not updated yet:
![Screen Shot 2022-10-18 at 10 41 50 AM](https://user-images.githubusercontent.com/8001765/196505437-1ba26790-de9b-4bd6-abb8-a8d5c5fb11ed.png)

I think you're good to ship this how it is, but also we are very close to being able to eliminate the `get 'docs/*path'` rule in the future.

Reply:
O, good call! I'll add this to the backlog.